pom.xml

```diff
@@ -21,7 +21,7 @@
         <!-- Prototyping framework -->
 
         <dependency>
-            <groupId>com.github.ONSdigital</groupId>
+            <groupId>com.github.onsdigital</groupId>
```

Comment:
NB. Just noticed these changes got checked in. I changed these when working on zebedee dependency upgrades due to IntelliJ not liking the Mac case-insensitive filesystem. It doesn't hurt and actually improves things so would prefer to leave it like this if possible.